src/poprox_recommender/handler.py

```diff
@@ -1,5 +1,7 @@
 import base64
 import logging
+from transformers import AutoTokenizer, AutoModel
```

Comment:
I left a few comments on the changes in `handler.py`, which generally look good. I'd like to keep `handler.py` limited to the code that's processing the request and response though, so I'd suggest moving these to a new file. Maybe `topics.py`?